docs/prow/prow-monitoring.md

```diff
@@ -91,16 +91,19 @@ Follow these steps to save the dashboard:
 
 1. Export the dashboard to a JSON format.
 
-2. Save the JSON file under `prow/cluster/resources/monitoring/dashboards/`.
-
-3. Update the Grafana configuration on the cluster.
+2. Save the JSON file under `prow/cluster/components/monitoring/dashboards/` and create configmap out of it.
```

Comment:
```suggestion
2. Save the JSON file under `prow/cluster/components/monitoring/dashboards/` and create a ConfigMap out of it.
```

Reply:
In prow-monitoring.md document we repetitively use `configmap` instead of `ConfigMap`. Now we will have two versions of configmaps in one document :). I will fix all other configmaps in that file.

Reply:
> In prow-monitoring.md document we repetitively use `configmap` instead of `ConfigMap`. Now we will have two versions of configmaps in one document :). I will fix all other configmaps in that file.

Good decision ;) See: https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#use-bold-font-for-these-items

Reply:
it is ready :)